tests/proofs/dexter/dexter-spec.md

```diff
@@ -124,6 +124,29 @@ If any of the conditions are not satisfied, the call fails.
       orBool LQTAddress =/=K #Address("tz1Ke2h7sDdakHJQh8WX4Z372du1KChsksyU")
 ```
 
+## Default
+
+Adds more money to the xtz reserves if the following conditions are satisifed:
+
+1.  the token pool is _not_ currently updating (i.e. `storage.selfIsUpdatingTokenPool = false`)
+
+```k
+  claim <k> #runProof(_IsFA2, Default) => . </k>
+        <stack> .Stack </stack>
+        <selfIsUpdatingTokenPool> false </selfIsUpdatingTokenPool>
+        <myamount> #Mutez(Amount) </myamount>
+        <xtzPool> #Mutez(XtzPool => XtzPool +Int Amount) </xtzPool>
+     requires #IsLegalMutezValue(XtzPool +Int Amount)
```

Comment:
I guess technically the `Amount` must also be >=  0, but maybe that is ensured some other way through the semantics?

Reply:
Yes, that's a good point. It won't get handled in any other way. I will update the claim to ensure that.